woocommerce/woocommerce-ios
# [Bundle products in order] Add pricedIndividually property to ProductBundleItem in Networking/Yosemite layers

Part of: #11250
⚠️ Depends on #11314 ⚠️ 


## Why
In a product bundle, the bundled child items can either be included in the price of the bundle or be priced individually (their prices added to the bundle price). We want to be able to display the child item price differently during order creation depending on whether the child item is priced individually.

## How
This adds a `pricedIndividually` property to the `ProductBundleItem` networking model, and updates the Yosemite `ProductBundleItem+ReadOnlyConvertible` to handle the stored attribute added in #11314.

The UI changes to use this new property are made in a separate PR (#11321)

## Testing instructions
Confirm all tests pass. (We aren't handling the new attribute in the UI yet.)

You can also build and run the app as a confidence check, using the testing instructions in #11321.


---
